documentation/2.0/content/rcuinfo.md

```diff
@@ -161,3 +182,205 @@ domainInfo:
+                            Value: 'true'
+```
+
+
+#### Oracle Active Grid Link Data Source (AGL)
+
+For setting data source to access Oracle Active Grid Link database, besides updating the `URL`, `PasswordEncrypted`, and the `user` property value; 
+you can specify additional `JDBCOracleParams` under `JdbcResource` of each data source.  For example,
+
+```yaml
+      mds-owsm:
+            Target: cluster-1,admin-server
+            JdbcResource:
+              JDBCOracleParams:
+                OnsNodeList:  'node1, node2'
+                ....
+```
+
+You can run WDT tool `modelHelp.sh -oracle_home <oracle home> resources:/JDBCSystemResource/JdbcResource/JDBCOracleParmas` for the complete list 
+of fields.
+
+#### Multi Data Sources
+
+If your database does not support high availability natively such as RAC or AGL, then WebLogic provides a Multi Data Source option, which basically the main data source is a logical data source that
+contains multiple physical data sources.   In order to create a spare model,  you can for each data source
+
+1. Update the templated data source `URL`, `EncryptedPassword`, and 'user' property value.  In the `URL` part, you can specify one of the accessible physical database URL.
+2. Duplicate each datasource two times
+3. Change the duplicated data source name, `JNDIName` by appending `-1`, `-2` etc. The `-1`, `-2` data sources are the physical data source, you will need to update their `URL` too.
+
+For example (note: details replaced by `....` for easier reading),
+
+```yaml
+       opss-data-source:
+           ....
+                JDBCDriverParams:
+                    URL: jdbc:oracle:thin://@somewhere:1521/db-node1
+                    PasswordEncrypted:  'actualpassword`
+           .... 
+       'opss-data-source-1':
+           ....
+               JDBCDataSourceParams:
+                 GlobalTransactionsProtocol: None
+                 JNDIName: jdbc/OpssDataSource-2
+                JDBCDriverParams:
+                    URL: jdbc:oracle:thin://@somewhere:1521/db-node1
+                    PasswordEncrypted:  'actualpassword`
+           ....
+       'opss-data-source-2':
+          ....
+           JDBCDataSourceParams:
+             GlobalTransactionsProtocol: None
+             JNDIName: jdbc/OpssDataSource-2
+           JDBCDriverParams:
+             URL: jdbc:oracle:thin://@somewhere:1521/db-node2
+             PasswordEncrypted:  'actualpassword`
+             DriverName: oracle.jdbc.OracleDriver
+           ....
+```
+
+
+4. Update the original data source to include the physical data source list.
+
+For example,
+
+```yaml
+       opss-data-source:
+           ....
+            JDBCDataSourceParams:
+              DataSourceList:  'opss-data-source-1, opss-data-source-2'
+              AlgorithmType:  'FailOver'
+           ....
+```
+
+You can run WDT tool `modelHelp.sh -oracle_home <oracle home> resources:/JDBCSystemResource/JdbcResource/JDBCDataSourceParams` for the complete list
```

Comment:
re-write: For the complete list of fields, run the WDT command, `modelHelp.sh -oracle_home <oracle home> resources:/JDBCSystemResource/JdbcResource/JDBCOracleParmas`.